grow-website-gatsby/src/pages/components/People/People.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { Fragment } from "react"
+import * as styles from "../../Index.module.scss"
+
+import PeopleData from "./PeopleData"
+
+const People = () => {
+  return (
+    <Fragment>
+      <div id="people" className={styles.headingLarge}>
+        People
+      </div>
+      <div className={styles.uMarginBottomML}>
+        <div className={styles.uWidthSixtyPercent}>
+          {PeopleData.description.map((paragraph, i) => {
+            return (
+              <p className={styles.paragraphSmall} key={i}>
+                {paragraph}
+              </p>
+            )
+          })}
+        </div>
+      </div>
+      <div className={styles.uMarginBottomL}>
+        <div className={styles.grid}>
+          {PeopleData.people.map((person, i) => {
+            return (
+              <div className={styles.gridItemContainer} key={i}>
```

Comment:
@catepalmer arguably you could make a `Person` component, but I think this looks much better than before :) . Good job!